src/main/java/org/mutabilitydetector/checkers/MutableTypeToFieldChecker.java

```diff
@@ -169,9 +174,9 @@ private void checkIfClassIsMutable(FieldInsnNode fieldInsnNode, Type typeAssigne
                     setAssigningToGenericFieldResult(fieldName, fieldLocation);
                     break;
                 }
-
-                MutabilityLookup mutabilityLookup = mutableTypeInfo.resultOf(dotted(ownerClass), assignedToField, analysisInProgress);
-
+                
+                MutabilityLookup mutabilityLookup = mutableTypeInfo.resultOf(dotted(ownerClass), assignedToField,analysisInProgress);
+                
```

Comment:
I'm really only seeing whitespace changes here (and in a bunch of other places still):smile: